install/tests/Makefile

```diff
@@ -314,6 +314,9 @@ self-signed-config:
 	envsubst < ./manifests/kots-config-self-signed.yaml > tmp_2_config.yml
 	yq m -i tmp_config.yml tmp_2_config.yml
 
+	# upload the Custom CA Cert into tf-state
```

Comment:
Can you also add commands to delete this file upon cleanup? It can be i[n this target ](https://github.com/gitpod-io/gitpod/pull/13305/files#diff-bd7b7f3a397df93111a6b6e2a4284b01bd57e04474bc9f91f1b02ee1960399a2L448) if you would like.

Reply:
I've added it to `destroy-kubeconfig`, Do you think we should do it separately? 🤔